tests/posix_paths/test_posix_paths.py

```diff
@@ -0,0 +1,205 @@
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import pytest
+
+import pystac
+from tests.conftest import get_data_file
+from tests.utils import ARBITRARY_BBOX, ARBITRARY_EXTENT, ARBITRARY_GEOM
+
+
+def check_link(link: Optional[pystac.Link]) -> None:
+    assert link is not None
+    href = link.get_target_str()
+    assert href is not None
+    assert "\\" not in href
+    print()
+    print(href)
```

Comment:
Shouldn't have leftover `print`s in tests.

Reply:
Ugh. Such slop. Nuked. Thanks for catching that.